QTeachersApp/View/create.cpp

```diff
@@ -0,0 +1,99 @@
+
+    }
+
+    Create::~Create()
+    {
+        delete ui;
+    }
+
+    void Create::on_pushButton_clicked()
+    {
+        // Getting data from lineEdits of ui - START OF CODE
+        QString name, surname, sex, pesel, dateOfBirth, title, listOfSubjects;
+        name = ui->lineEdit_name->text();
+        surname = ui->lineEdit_surname->text();
+        sex = ui->lineEdit_sex->text();
+        pesel = ui->lineEdit_pesel->text();
+        dateOfBirth = ui->lineEdit_dateOfBirth->text();
+        title = ui->lineEdit_title->text();
+        listOfSubjects = ui->lineEdit_listOfSubjects->text();
+        // Getting data from lineEdits of ui - END OF CODE
+        if((*myTeachersDatabase).isOpen())
+        {
+            qInfo("I'm in creation mode right now!");
+            // Checking if there is the same data in database already - START OF CODE
+            QSqlQuery querySelect;
+            querySelect.prepare("SELECT * FROM Teachers");
+            querySelect.exec();
+            bool checkForEqualisation = false; // if the same data (with only diffrence being another "ID") are already in teachers.sqlite database
+            while(querySelect.next())
+            {
+                   if(name == querySelect.value(1).toString()
+                      && surname == querySelect.value(2).toString()
+                      && sex == querySelect.value(3).toString()
+                      && pesel == querySelect.value(4).toString()
+                      && dateOfBirth == querySelect.value(5).toString()
+                      && title == querySelect.value(6).toString()
+                      && listOfSubjects == querySelect.value(7).toString())
+                    {
+                        checkForEqualisation = true;
+                        qDebug() << "Creating your new \"teacher\" will be imposssible, because he already is in our database (only with another id).";
+                    }
+
+            }
+            if(!dataValidation(name, surname, sex, pesel, dateOfBirth, title, listOfSubjects) || checkForEqualisation)
+            {
+
+                clearingLineEdits();
+                return;
+            }
+            // Checking if there is the same data in database already - END OF CODE
+            // Creation of new teacher - START OF CODE
+            QSqlQuery queryCreate;
+            queryCreate.prepare("INSERT INTO Teachers (Name, Surname, Sex, PESEL, DateOfBirth, Title, ListOfSubjects) VALUES (?, ?, ?, ?, ?, ?, ?)");
+            queryCreate.addBindValue(name);
+            queryCreate.addBindValue(surname);
+            queryCreate.addBindValue(sex);
+            queryCreate.addBindValue(pesel);
+            queryCreate.addBindValue(dateOfBirth);
+            queryCreate.addBindValue(title);
+            queryCreate.addBindValue(listOfSubjects);
+            if(queryCreate.exec())
+            {
+                qInfo() << "Everything is working and you querry was added to your teachers database. :)";
```

Comment:
Wydaje mi się, że qInfo nie wyświetla żadnego info (albo wyświetla je w konsoli, która nie będzie widoczna dla usera). Dodaj jakiś modal, który będzie reagował na to i pokazywał tekst użytkownikowi.